site/content/resources/_index.html

```diff
@@ -109,5 +109,23 @@ <h2>Resources about Pinniped, such as videos, podcasts, and community articles</
             </div>
         </div>
 
+        <div class="col">
+            <a href="https://aws.amazon.com/blogs/opensource/simplify-amazon-eks-multi-cluster-authentication-with-open-source-pinniped/"
+               target="_blank">
+                <div class="icon">
+                    <img src="https://d2908q01vomqb2.cloudfront.net/ca3512f4dfa95a03169c5a670a4c91a19b3077b4/2023/06/13/Simplify-EKS-multicluster-authentication-main-art-1024x513.png"
+                         width="311px"/>
```

Comment:
This image source is very large. Is there a better way to constrain its size to match the size of the other tiles on this page? This "works" except when you make your browser very narrow and the page responds by laying out the tiles vertically, then the width doesn't quite look right.

Reply:
Uhhhh to be honest I'm not sure. It doesn't look great in smallscreen but it's useable. I'll approve.